Automattic/wp-calypso
# Docs: Use HEAD rather than master for GitHub URIs

#### Changes proposed in this Pull Request

* Change all GitHub URIs that reference `blob/master` to use `blob/HEAD` which is a default, branch-name-agnostic alias for the "main" branch of a repository, regardless of what it is actually called.

Note: Sometimes this alias does not exist if you do some funky stuff but it is the neutral default when you create a new repository through GitHub's UI. When the alias does not exist, the only thing we can do is wait for the project to change their default branch name, if they ever do. I manually checked each repository we reference and didn't find anything other than truly dead links (to files that no longer exist).

#### Testing instructions

* Ensure all changed links still work (you only need to test one link per repository as really it's just necessary to verify that the repo configuration includes HEAD -> master alias).

Part of https://github.com/Automattic/wp-calypso/issues/43395
